_sass/desktop.scss

```diff
@@ -42,14 +42,14 @@ section.code {
 }
 
 section.instructions {
-  section {
+  > section {
```

Comment:
ich. Do we have to? Can we add class names to the sections? This feels like a move to a sunset.